src/auto_scheduler/feature.cc

```diff
@@ -233,14 +233,9 @@ AnnotationPosType GetAnnotationPosEncoding(const Var& var, const Array<PrimExpr>
   }
 }
 
-// Return the extent of a for loop
-int64_t GetLoopExtent(const ForNode* node) {
-  auto pint = node->extent.as<IntImmNode>();
-  if (pint != nullptr) {
-    return pint->value;
-  } else {
-    return 1;
-  }
+// Return the maximum extent of a for loop
+int64_t GetLoopExtent(const ForNode* node, const Analyzer& ana) {
```

Comment:
NOTE: this is a change of behavior that might worth checking, const_int bound may return INT_MAX for cases that it cannot analyze

Reply:
I've added a check that return 1 if const_int_bound returns inf (it way of saying it cannot analyze).

Reply:
This LGTM. although I do not expect regression in this case,  would be good to run some tests to confirm, not a blocker to merge though. cc @junrushao1994